invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json

```diff
@@ -381,12 +348,55 @@
       },
       "subject": {
         "type": "string"
+      },
+      "scheme": {
+        "description": "Identifier of the subject scheme.",
+        "$ref": "local://definitions-v1.0.0.json#/identifier"
+      },
+      "props": {
+        "type": "object",
+        "patternProperties": {
+          "^.*$": {
```

Comment:
Since this is allowing any property, why is this needed?

Reply:
Just for defining the constraint that the values for those keys must be strings, as we are storing parents as a string.
the subject hierarchy is a list but it is not stored anywhere just dumped when indexing.